assets/src/dashboard/app/api/useTemplateApi.js

```diff
@@ -57,12 +58,41 @@ export function reshapeTemplateObject(isLocal) {
   });
 }
 
+// TODO once templates are all connected to an API this and the above reshapeTemplateObject should be able to be one and the same
+// Connecting the ability to create a template from a story I wanted to be able to see the results on saved templates, this endpoint is still missing some template related data.
+export function reshapeSavedTemplates({
+  author,
+  isLocal = true,
+  id,
+  title,
+  modified,
+  tags = [],
+  colors = [],
+  createdBy = 'N/A',
+  description,
+  pages,
+}) {
+  return {
+    author,
+    isLocal,
+    id,
+    title: title.rendered,
+    createdBy,
+    description,
+    status: 'template',
+    modified: moment(modified),
+    tags,
+    colors,
+    pages,
+    centerTargetAction: false, // `${APP_ROUTES.TEMPLATE_DETAIL}?id=${id}&isLocal=${isLocal}`,
```

Comment:
blocked this off for the time being because the necessary data to render the detail view isn't present and i didn't want to go through the detail view in this PR making data optional since that's out of scope and the saved templates view isn't in use yet.